docs/architecture/cloud-native/index.md

```diff
@@ -41,33 +41,35 @@ Author:
 >
 > **Rob Vettor** - Microsoft - Principal Cloud System Architect/IP Architect - [RobVettor.com](http://robvettor.com)
 
-Editors:
-
-> **John Doe**
-
-## Introduction
+Participants and Reviewers:
 
-TODO
+> **Cesar de la Torre**, Sr. PM, .NET product team, Microsoft Corp.
```

Comment:
Can we be consistent if we're going to refer to Microsoft with or without Corp?

Reply:
```suggestion
> **Cesar de la Torre**, Principal Program Manager, .NET team, Microsoft
```